oshepherd/api/embeddings/routes.py

```diff
@@ -0,0 +1,43 @@
+"""
+Generate embeddings
+API implementation of `POST /api/embeddings` endpoint, handling embeddings orchestration, as replica of the same Ollama server endpoint.
+Ollama endpoint reference: https://github.com/ollama/ollama/blob/main/docs/api.md#generate-embeddings
+"""
+
+import time
+from flask import Blueprint, request
+from oshepherd.api.utils import streamify_json
+from oshepherd.api.embeddings.models import EmbeddingsRequest
+
+embeddings_blueprint = Blueprint("embeddings", __name__, url_prefix="/api/embeddings")
+
+
+@embeddings_blueprint.route("/", methods=["POST"])
+def embeddings():
+    from oshepherd.worker.tasks import exec_completion
+
+    print(f" # request.json {request.json}")
```

Comment:
`print` found

Reply:
Yea, lets improve these with proper global logger later when we tackle abstraction 😉